opta/flyte.yaml

```diff
@@ -0,0 +1,161 @@
+environments:
+  - name: default
+    path: "./env.yaml" # NOTE: relative path to environment
+    variables:
+      region: {region}
+      account_id: {account_id}
+name: service-flyte
+modules:
+  - name: postgres
+    type: aws-postgres
+  - name: s3
+    type: aws-s3
+    bucket_name: "{parent_name}-{layer_name}"
+  - name: notifcationsQueue
+    type: aws-sqs
+  - name: schedulesQueue
+    type: aws-sqs
+  - name: topic
+    type: aws-sns
+    sqs_subscribers:
+      - "${{module.notifcationsQueue.queue_arn}}"
+  - name: adminflyterole
+    type: aws-iam-role
+    extra_iam_policies:
+      - "arn:aws:iam::aws:policy/CloudWatchEventsFullAccess"
+#      - "arn:aws:iam::{vars.account_id}:policy/{env}-{env}-awsses-sender" # Uncomment out for SES
+    allowed_k8s_services:
+      - namespace: "*"
+        service_name: "*"
+    links:
+      - s3: [ "write" ]
+      - notifcationsQueue
+      - schedulesQueue
+      - topic
+  - name: userflyterole
+    type: aws-iam-role
+    extra_iam_policies:
+      - "arn:aws:iam::aws:policy/CloudWatchEventsFullAccess"
+#      - "arn:aws:iam::{vars.account_id}:policy/{env}-{env}-awsses-sender" # Uncomment out for SESre, change the templating
```

Comment:
```suggestion
```

Reply:
Why do you want to remove this part?

Reply:
user role doesn't need such permission... it only needs the s3 read/write access...